torch/testing/_internal/common_methods_invocations.py

```diff
@@ -63,6 +63,10 @@ def __init__(self,
                  dtypesIfCUDA=None,  # dtypes this function is expected to work with on CUDA
                  dtypesIfROCM=None,  # dtypes this function is expected to work with on ROCM
                  test_inplace_grad=True,  # whether to gradcheck and gradgradcheck the inplace variant
+                 is_autodiffed=False,  # if a op's aten::node is expected to be symbolically autodiffed
+                 autodiff_nonfusible_nodes=None,  # ex: ['aten::add', 'aten::mm'], default is populated to be ['aten::name']
```

Comment:
Add the expected type and explain how these are used, here and below.

For example, "a list of strings with node names that should not appear in any fusion group when this operation is fused, ex: ['aten::add', 'aten::mm']. If none, defaults to ['aten::<name of operator>'].